### ISSUE TITLE
Rar files being symlinked

### ISSUE THREAD

**Issue by eoghan2t9** [sureshfizzy/CineSync#2]
Would it be possible for the script to skip creating symlinks for rars and spilt rars I know it won't affect the files being played but would clean up the folders better.

**Comment by sureshfizzy** [comment]
I'll check on this, so what's the current behaviour when rar files hits ?

**Comment by eoghan2t9** [comment]
That was quick thanks very much 😀

**Comment by sureshfizzy** [comment]
> That was quick thanks very much 😀

Welcome :)

**Comment by eoghan2t9** [comment]
Think you might need to tweak the script some more lol some spilt rar have the extension .r1 .r2 and so on. Also would it be possible to do the same for movies or would that be too much work lol

**Comment by sureshfizzy** [comment]
> Think you might need to tweak the script some more lol some spilt rar have the extension .r1 .r2 and so on. Also would it be possible to do the same for movies or would that be too much work lol

I came accross only rar file for now, drop me the list of unwanted extensions and I'll filter them out. And for movies I'm looking forward for it since the current implementation won't accept them.

**Comment by eoghan2t9** [comment]
Fair enough I wonder if there is a way of checking with APIs from tmdb for the title and match it that way. As for the rars this is what I found online

The first file used the extension .rar, then .r00 for the second, and then .r01, .r02, and so on.

If you can also once the rars have been filtered would it be possible on the next run of the script that it removes already symlinks rars out of the shows folder?

**Comment by sureshfizzy** [comment]
> Fair enough I wonder if there is a way of checking with APIs from tmdb for the title and match it that way. As for the rars this is what I found online
> 
> The first file used the extension .rar, then .r00 for the second, and then .r01, .r02, and so on.
> 
> If you can also once the rars have been filtered would it be possible on the next run of the script that it removes already symlinks rars out of the shows folder?

Tmdb API was my second plan for renaming files, I need to look into it further.

Also, a check for removing previously symlinked rar files is possible to implement.

**Comment by eoghan2t9** [comment]
Absolutely fantastic mate thanks also I think the real time monitoring isn't working or I've done something wrong the main script runs fine and does the links if I do a full scan however the realtime scan seems to run but exit far too quickly. I'll have a look at my config before making another bug report 😉

**Comment by sureshfizzy** [comment]
> Absolutely fantastic mate thanks also I think the real time monitoring isn't working or I've done something wrong the main script runs fine and does the links if I do a full scan however the realtime scan seems to run but exit far too quickly. I'll have a look at my config before making another bug report 😉

Do open a new report if something hits different in that. Also your rar files will be handled smoothly now, do pull the latest code.

**Comment by eoghan2t9** [comment]
Your a genius mate. Is there any debugging in the python script for the realtime monitor as it's definitely exiting to quickly?

**Comment by sureshfizzy** [comment]
Get inside Scripts folder and try to run Realtime script directly. You will find what is causing issue.

**Comment by eoghan2t9** [comment]
Tried that and it says its completed the run but nothing seems to get picked up by plex. When i rerun the cinesync.sh then it will added the symlinks to be picked up

**Comment by sureshfizzy** [comment]
> Tried that and it says its completed the run but nothing seems to get picked up by plex. When i rerun the cinesync.sh then it will added the symlinks to be picked up

Try this,

1) pick a new file and wait for about 2-3 minutes 

2) during 3rd or 4th cycle of real time monitor, you will get those files.

Basically that script will be running for scan every 60
